frontend/src/app/libs/auth/models/userCredentials.ts

```diff
@@ -0,0 +1,4 @@
+export class UserCredentials {
+	public email?: string;
+	public password?: string;
```

Comment:
update file naming to follow naming convention (user-credentials.ts)
Also please avoid using class for interface purposes.